modules/mono/glue/cs_files/Rect2.cs

```diff
@@ -11,63 +11,68 @@ namespace Godot
     [StructLayout(LayoutKind.Sequential)]
     public struct Rect2 : IEquatable<Rect2>
     {
-        private Vector2 position;
-        private Vector2 size;
+        private Vector2 _position;
+        private Vector2 _size;
 
         public Vector2 Position
         {
-            get { return position; }
-            set { position = value; }
+            get { return _position; }
+            set { _position = value; }
         }
 
         public Vector2 Size
         {
-            get { return size; }
-            set { size = value; }
+            get { return _size; }
+            set { _size = value; }
         }
 
         public Vector2 End
         {
-            get { return position + size; }
+            get { return _position + _size; }
         }
 
         public real_t Area
         {
             get { return GetArea(); }
         }
 
+        public Rect2 Abs()
+        {
+             return new Rect2(Vector2.Zero, _size.Abs());
```

Comment:
Why is the absolute value of a Rect2 positioned at zero?

Reply:
Good spot.  Looking back it seems I misread the docs:

> Returns a `Rect2` with equivalent position and area, modified so that the top-left corner is the origin and `width` and `height` are positive.

I will fix it.

Reply:
The docs are misleading, but I see what they mean. If a size in one direction is negative, add that to the origin and set the size to the absolute value so that it's the same rectangle just with the start position at the top-left.

Reply:
This should be fixed now.